content/en/opentelemetry/instrument/api_support/dotnet/logs.md

```diff
@@ -0,0 +1,90 @@
+---
+title: .NET OpenTelemetry Logs Support
+code_lang: logs
+type: multi-code-lang
+code_lang_weight: 2
+further_reading:
+    - link: opentelemetry/correlate/logs_and_traces
+      tag: Documentation
+      text: Correlate OpenTelemetry Traces and Logs
+---
+
+## Overview
+
+You can export OTLP logs from your .NET application using the Datadog SDK (`dd-trace-dotnet`).
+
+This feature works by intercepting logs from the built-in `Microsoft.Extensions.Logging.ILogger` library and exporting them as structured OTLP data.
+
+<div class="alert alert-info">This implementation does not use the OpenTelemetry Logs API. It is not compatible with the <code>OpenTelemetry.SDK.Logs</code> or <code>OpenTelemetry.Exporter.Otlp.Logs</code> packages. You must remove them to use this feature</div>
+
+## Prerequisites
+
+- **Datadog SDK**: `dd-trace-dotnet` version [x.y.z] or later.
+- **An OTLP-compatible destination**: You must have a destination ready to receive OTLP data, such as the Datadog Agent or OpenTelemetry Collector.
+
+## Setup
+
+1.  Install the Datadog SDK:
+    ```sh
+    dotnet add package Datadog.Trace
+    ```
+2.  Enable OTel logs export by setting the following environment variable:
+    ```sh
+    export DD_LOGS_OTEL_ENABLED=true
+    ```
+3.  Ensure Datadog instrumentation is active for your application.
+    ```sh
+    [TODO: Code example]
+    ```
+
+## Examples
+
+### Standard logging
+
+```csharp
+[TODO: Code example]
+```
+
+### Trace and log correlation
+
+```csharp
+[TODO: Code example]
+```
+
+## Supported configuration
+
+To enable this feature, you must set `DD_LOGS_OTEL_ENABLED=true`.
+
+All OTLP exporter settings (such as endpoints, protocols, and timeouts), resource attributes, and batch processor settings are configured using a shared set of OpenTelemetry environment variables.
+
+For a complete list of all shared OTLP environment variables, see [OpenTelemetry Environment Variables Interoperability][1].
+
+## Migrate from other setups
+
+### Existing OTel setup (manual configuration)
+
+This Datadog feature is not compatible with applications already using the OpenTelemetry Logs API or OpenTelemetry SDK for logs.
```

Comment:
```suggestion
This Datadog feature is not compatible with applications already using the OpenTelemetry Logs API or OpenTelemetry SDK for logs.
```
This seems like a fairly important callout, like if you are using X or Y, do not try these steps, may be worth converting to a Note callout ?